client/src/routes/game/+page.svelte

```diff
@@ -28,7 +31,7 @@
       return;
     }
 
-    const lobbyDocRef = doc(lobbyCollection, lobbyCode);
+    lobbyDocRef = doc(lobbyCollection, lobbyCode);
```

Comment:
why is this changed from const to let?

Reply:
I used lobbyDocRef in an export let for my own WinLoss.svelte component so I had to change that